### ISSUE TITLE
flow is busy: [processing] /Out of retries, exiting!

### ISSUE THREAD

**Issue by Gozala** [facebook/flow#1053]
Lately I keep getting these errors, when running `flow status`. Here is also an output in the log:

```
[2015-11-06 13:31:20] Initializing Server (This might take some time)
[2015-11-06 13:31:20] executable=/usr/local/lib/node_modules/flow-bin/vendor/flow
[2015-11-06 13:31:20] version=0.18.1
[2015-11-06 13:31:22] Parsing
[2015-11-06 13:31:46] Running local inference
[2015-11-06 13:32:17] Merging
[2015-11-06 13:32:22] Server is READY
[2015-11-06 13:32:22] Took 61.862587 seconds to initialize.
Status: Error
recheck 1 files:
1/1: /Users/gozala/Projects/browser.html/src/browser/updater.js
[2015-11-06 13:32:32] Running local inference
remerge 1 dependent files:
1/1: /Users/gozala/Projects/browser.html/src/browser/browser.js
[2015-11-06 13:32:32] Merging
Status: Error
recheck 1 files:
1/1: /Users/gozala/Projects/browser.html/src/browser/updater.js
[2015-11-06 13:32:37] Running local inference
remerge 1 dependent files:
1/1: /Users/gozala/Projects/browser.html/src/browser/browser.js
[2015-11-06 13:32:37] Merging
Status: Error
Status: Error
recheck 1 files:
1/1: /Users/gozala/Projects/browser.html/src/browser/updater.js
[2015-11-06 13:33:15] Running local inference
remerge 1 dependent files:
1/1: /Users/gozala/Projects/browser.html/src/browser/browser.js
[2015-11-06 13:33:16] Merging
Status: OK
Status: OK
recheck 1 files:
1/1: /Users/gozala/Projects/browser.html/src/browser/updater.js
[2015-11-06 13:33:24] Running local inference
remerge 1 dependent files:
1/1: /Users/gozala/Projects/browser.html/src/browser/browser.js
[2015-11-06 13:33:25] Merging
```

**Comment by Gozala** [comment]
I can also provide specific change that triggers this issue, but without a full context it may not be as useful. Alternatively I can also provide a link to a tree that can be used to reproduce this.

**Comment by Gozala** [comment]
Also once flow server runs into this problem there is no way to recover other than killing it.

**Comment by rileylark** [comment]
We are hitting this too - have you had any luck working around this?

**Comment by ivawzh** [comment]
I got the same error

**Comment by mroch** [comment]
this is likely fixed by 7989bdca91aa8026a3cb7e78592de96bb7f59008, which will be in 0.19.

**Comment by samwgoldman** [comment]
Can't repro, and we've done considerable work on this front. If you're getting this error and found this issue via search, please open a new issue and if at all humanly possible provide repro code (pretty pretty please).
